app/harvesters/nara_harvester.rb

```diff
@@ -0,0 +1,126 @@
+##
+# A harvester for NARA's API
+#
+# @see Krikri::Harvesters::ApiHarvester
+class NaraHarvester < Krikri::Harvesters::ApiHarvester
+  DEFAULT_URI = 'https://catalog.archives.gov/api/v1'
+  DEFAULT_NAME = 'nara'
+  DEFAULT_BATCHSIZE = 10
+  DEFAULT_ID_FILENAME = '/var/tmp/nara_ids'
```

Comment:
I'm +0 on moving this to configuration. The default can already be overridden in an instance, and it seems like a lot of overhead.

I think I might be in favor of moving it to `RAILS_ROOT/storage`, though?